tests/components/main-panel/main-panel.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import { render, screen } from "@testing-library/react";
+import "jest-canvas-mock";
+import { MainPanel } from "src/components";
+import { EElementType } from "src/context-providers";
+import { ELEMENT_BUTTON_LABELS } from "src/data/elements-data";
+import { TestHelper, mockBuilderState } from "src/util/test-helper";
+
+describe("main-panel", () => {
+    afterEach(() => {
+        jest.restoreAllMocks();
+        jest.resetAllMocks();
+    });
+
+    describe("conditional rendering of main panel content", () => {
+        it("should display the empty panel content", async () => {
+            renderComponent();
+            await expect(getEmptyPanelContent()).resolves.toBeInTheDocument();
```

Comment:
I think can just get directly instead of having a function for it. I dont foresee any other test cases at the moment